draft-ietf-masque-connect-udp.md

```diff
@@ -292,6 +292,31 @@ reduce performance as both can sometimes independently retransmit the same
 data. To avoid this, HTTP/3 datagrams SHOULD be used.
 
 
+## Tunneling of ECN Marks
+
+CONNECT-UDP does not create an IP-in-IP tunnel, so the guidance in {{?RFC6040}}
+about transferring ECN marks between inner and outer IP headers does not apply.
```

Comment:
There is no inner IP header in CONNECT-UDP, could we rephrase this sentence to make that clearer?